command/cp.go

```diff
@@ -287,7 +293,9 @@ func (c Copy) Run(ctx context.Context) error {
 			continue
 		}
 
-		if object.StorageClass.IsGlacier() {
+		// if force glacier transfer flag is set, it will omit check for the if block
+		// if not set, then it will check whether the object is glacier.
+		if !c.forceGlacierTransfer && object.StorageClass.IsGlacier() {
```

Comment:
```suggestion
		if object.StorageClass.IsGlacier() && !c.forceGlacierTransfer {
```

IMHO it's easier on the eyes when the checks are swapped and the checks are pretty self explanatory making comment redundant.

Reply:
changed the order of conditions and removed the comment lines.